docs/data-sources/cluster.md

```diff
@@ -82,6 +82,9 @@ Read-Only:
 
 Read-Only:
 
+- `disk_iops` (Number)
+- `disk_size_gb` (Number)
+- `num_cores` (Number)
```

Comment:
Wait why are these "read only"?

Reply:
I'm unsure, but maybe the documentation autogenerates `Read-Only` for `Computed` fields? cc: @posriniv @cdavid are you aware?

Reply:
@daniel-yb  These are readonly because it is a datasource